vision.md

```diff
@@ -1,52 +1,41 @@
-Stated simply...
+## Stated simply...
 
-**NUnit 3 is a world-class platform for individuals and teams using a test-based approach to .NET software development.**
+**NUnit aims to be a best in breed test framework that enables end to end testing of .NET projects by individuals and teams. It is unopinionated and supports a number of different kinds of tests. NUnit allows for alternative approaches to testing and is usable by team members in a variety of roles.**
 
```

Comment:
Sorry - I explained this badly!

The primary point of my suggestion was combining "supports a number of different kinds of tests" and "allows for alternative approaches to testing" into one phrase (I chose "supports a number of different approaches to testing").

Merging the sentences after that was just the next natural step, now the content was shortened.

To me - whilst the two different phrases make sense with examples, they seemed a little ambiguous on first reading. As above, however, perhaps that's just my non-academic approach to testing! I'll go with either.